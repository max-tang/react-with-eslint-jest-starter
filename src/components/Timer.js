import React from 'react';

const Timer = () => (
  <div>A timer</div>
);

export default Timer;
// export default Timer extends Component {
//   render() {
//     return (
//       <div>A timer</div>
//     );
//   }
// }
