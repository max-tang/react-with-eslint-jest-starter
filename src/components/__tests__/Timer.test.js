import React from 'react';
import renderer from 'react-test-renderer';
import Timer from '../Timer';

test('Timer component works', () => {
  const component = renderer.create(
    <Timer />,
  );
  const timerJson = component.toJSON();
  expect(timerJson).toMatchSnapshot();
});
