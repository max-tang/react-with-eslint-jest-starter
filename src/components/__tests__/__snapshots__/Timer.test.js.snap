// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component works 1`] = `
<div>
  A timer
</div>
`;
